docs/references/users/update-user-labels.md

```diff
@@ -0,0 +1 @@
+Update the user labels by its unique ID. The array you pass is stored as is, and replaces any previous value. Each label has a maximum of 128 characters.
```

Comment:
I would also add another line explaining where this can come useful and how it connect to the permissions system. Might also be useful to add another line for labels vs teams.

Reply:
@eldadfux are we adding the permission side of things as part of this release?